playground/mocks/data/idp/idx/user-unlock-account.json

```diff
@@ -0,0 +1,177 @@
+                                  "form": {
+                                      "value": [
+                                          {
+                                              "name": "id",
+                                              "required": true,
+                                              "value": "aut215n1MxHoGZdfd0g4",
+                                              "mutable": false
+                                          },
+                                          {
+                                              "name": "methodType",
+                                              "required": false,
+                                              "value": "email",
+                                              "mutable": false
+                                          }
+                                      ]
+                                  }
+                              },
+                              "relatesTo": "$.authenticatorEnrollments.value[0]"
+                          },
+                          {
+                              "label": "Phone",
+                              "value": {
+                                  "form": {
+                                      "value": [
+                                          {
+                                              "name": "id",
+                                              "required": true,
+                                              "value": "aut215oJJNVuOeUhN0g4",
+                                              "mutable": false
+                                          },
+                                          {
+                                              "name": "methodType",
+                                              "type": "string",
+                                              "required": false,
+                                              "options": [
+                                                  {
+                                                      "label": "SMS",
+                                                      "value": "sms"
+                                                  },
+                                                  {
+                                                      "label": "Voice call",
+                                                      "value": "voice"
+                                                  }
+                                              ]
+                                          }
+                                      ]
+                                  }
+                              },
+                              "relatesTo": "$.authenticatorEnrollments.value[1]"
+                          }
+                      ]
+                  },
+                  {
+                      "name": "identifier",
+                      "label": "Email or Username",
+                      "required": true
+                  },
```

Comment:
Shouldn't this be before the selection options? Because the text input needs to appear before the authenticator list?

Reply:
Nope. Doesn't matter.

Reply:
@zeusandrade-okta ^ FYI..